doc/FEATURES_RELEASE_2.md

```diff
@@ -41,6 +41,9 @@ DDI. Press `Submit`. The system should display the DDI information
 for the DDI found, including whether its disposition is `Good` or `Bad`,
```

Comment:
The problem with this is the consistency of things, his mock data refers to it as good/bad but our UI refers to it as Harmful/Not Harmful